### PR TITLE
I hate this

### PR DESCRIPTION
**Pull request by ghost**
Hopefully this fixes all my problems
Thank god I start programming tomorrow I might actually be able to understand what tf this all means
43B4AA37-1595-4AB2-B6E0-F3F6EA0BFB4A

### CODE REVIEW

File: app/assets/js/elements/skyclientStuff.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ class SkyclientStuff extends HTMLElement {
             class="block mx-auto bg-neutral-600 hover:bg-neutral-800 transition-all p-2 m-4 rounded-md"
             onclick="this.parentElement.close()"
           >
-            no thanks i am dum
+            I'll make my own modpack, and I know exactly what I'm doing.
```
Comment (KTibow) [inline app/assets/js/elements/skyclientStuff.js]: could you phrase this better

Comment (ghost) [reply app/assets/js/elements/skyclientStuff.js]: I’ll figure it out

Comment (ghost) [reply app/assets/js/elements/skyclientStuff.js]: how do I change it

Comment (ghost) [reply app/assets/js/elements/skyclientStuff.js]: brb gotta go to class

**Pull request merged**